test/e2e-tests.sh

```diff
@@ -65,14 +66,13 @@ function set_feature_gate() {
   kubectl patch configmap feature-flags -n tekton-pipelines -p "$jsonpatch"
 }
 
-function set_result_extraction_method() {
+function set_perfeatureflag() {
```

Comment:
why are you replacing result_extraction_method?
I think this should be a separate function.

Reply:
Sorry. I think I misread your comment. Please ignore.